eclass/kernel-build.eclass

```diff
@@ -203,6 +203,12 @@ kernel-build_src_configure() {
 			.config)
```

Comment:
The summary line of the commit message is too long (it wraps).

Reply:
I've reworded the summary line.